packages/x-charts/src/ChartContainer/useChartContainerProps.ts

```diff
@@ -29,15 +32,25 @@ export const useChartContainerProps = <
     colors,
     dataset,
     desc,
+    // @ts-ignore
     onAxisClick,
     disableAxisListener,
+    // @ts-ignore
     highlightedItem,
+    // @ts-ignore
     onHighlightChange,
     sx,
     title,
+    // @ts-ignore
     xAxis,
+    // @ts-ignore
     yAxis,
+    // @ts-ignore
     zAxis,
+    // @ts-ignore
+    rotationAxis,
+    // @ts-ignore
+    radiusAxis,
```

Comment:
Eh, these are bad 😅

Might as well have "any" as type. WDYT about moving these type changes to a different PR so we can discuss them in isolation? 🤔